internal/cmd/gocdk/add.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 The Go Cloud Authors
```

Comment:
Potentially n00b fly-by question: as far as naming / project organization, is it accepted practice to have all the code under "cmd"? Would the eventual situation have something like a package for the gocdk functionality and then a `main` package for the `gocdk` command at `cmd/gocdk.go` or some such? Is all of the code for `gocdk` going to be in `package main`?

Reply:
The only hard-ish requirement is that the base directory name align with the eventual binary name. I have seen `cmd/` used in a number of projects to separate out binaries from library packages: [Perkeep](https://perkeep.org/cmd/), [Kubernetes](https://godoc.org/k8s.io/kubernetes/cmd), [Juju](https://github.com/juju/juju/blob/develop/cmd/juju/main.go).